modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/Commenting/CommentingViewComponent.cs

```diff
@@ -98,11 +101,12 @@ private string GetCaptchaImageBase64(byte[] bytes)
     private async Task ConvertMarkdownTextsToHtml(CommentingViewModel viewModel)
     {
         viewModel.RawCommentTexts = new Dictionary<Guid, string>();
+        var referralLinks = viewModel.ReferralLinks?.ToArray();
```

Comment:
No need to convert the `IEnumerable<string>` to an array. 

```suggestion
        var referralLinks = viewModel.ReferralLinks.JoinAsString(" ");
```